contracts/0.4.24/lib/StakeLimitUtils.sol

```diff
@@ -0,0 +1,110 @@
+// SPDX-FileCopyrightText: 2022 Lido <info@lido.fi>
+
+// SPDX-License-Identifier: GPL-3.0
+
+/* See contracts/COMPILERS.md */
+pragma solidity 0.4.24;
+
+//
+// We need to pack four variables into the same 256bit-wide storage slot
+// to lower the costs per each staking request.
+//
+// As a result, slot's memory aligned as follows:
+//
+// LSB ------------------------------------------------------------------------------> MSB
+// 0______________________32______________128_________________________160______________256
+// |______________________|________________|___________________________|________________|
+// | prevStakeBlockNumber | prevStakeLimit | maxStakeLimitGrowthBlocks | maxStakeLimit  |
+// |<----- 32 bits ------>|<-- 96 bits --->|<---------- 32 bits ------>|<--- 96 bits -->|
+//
+//
+// NB: we represent `maxStakeLimitGrowthPeriod` as follows:
```

Comment:
Is this comment about `maxStakeLimitGrowthPeriod` is obsolete?

Reply:
yep, that was a typo, ty!